mne/time_frequency/tfr.py

```diff
@@ -29,52 +30,73 @@
 from ..externals.six import string_types
 
 
-def _get_data(inst, return_itc):
-    """Get data from Epochs or Evoked instance as epochs x ch x time"""
-    from ..epochs import _BaseEpochs
-    from ..evoked import Evoked
-    if not isinstance(inst, (_BaseEpochs, Evoked)):
-        raise TypeError('inst must be Epochs or Evoked')
-    if isinstance(inst, _BaseEpochs):
-        data = inst.get_data()
-    else:
-        if return_itc:
-            raise ValueError('return_itc must be False for evoked data')
-        data = inst.data[np.newaxis, ...].copy()
-    return data
+# Make wavelet
+
+@deprecated("This function will be removed in mne 0.14.")
+def morlet(sfreq, freqs, n_cycles=7.0, sigma=None, zero_mean=False):
```

Comment:
dpss was private but morlet wasn't, so for consistency. In the previous commit, I made both of them public, but I think @agramfort wanted to diminish the number of low level function related to time frequency transform. 

I have indeed missed this tutorials. I can put it back to public, although it seems a pretty low level function for MNE. It looks like more something relevant to scipy.signal no?